### DISCUSSION TITLE
Thank You!

### DISCUSSION THREAD

**Discussion by Sosukodo** [blend2d/blend2d#211] [General]
I just wanted to thank Petr Kobalicek and everybody else that has put work into the blend2d project :-)

I only recently became aware of this project and I am thoroughly impressed with everything it can do! I'm not one to gush (or even praise much) but I'm so happy this project is around. As soon as I saw the blend2d / SDL2 Texture example, I've been playing with it non-stop.

As soon as I actually make something interesting with it, I'll be supporting the project in other ways.

Thanks again!

**Comment by kobalicek** [comment]
Thanks that's great that you have found this project useful!
